src/codecs/cats/ldpc.c

```diff
@@ -0,0 +1,91 @@
+#include <assert.h>
+#include <string.h>
+
+#include "ldpc.h"
+#include "ldpc_matrices.h"
+
+#define GET_BIT(byte, bit) (((byte) & (1 << (7-(bit)))) != 0)
+#define SET_BIT(byte, bit) (byte |= 1 << 7-bit)
+#define CLEAR_BIT(byte, bit) (byte &= ~(1 << 7-bit))
+#define FLIP_BIT(byte, bit) (byte ^= (1 << (7 - bit)))
+
+cats_ldpc_code_t *cats_ldpc_pick_code(int len)
+{
+    if(128 <= len)
```

Comment:
Small thing, but let's try to stick with the formatting conventions:

- Spaces after keywords
- Always use braces `{}` and the way to use them (outside function definitions) is:

```
if (condition) {
  // TODO
} else {
  // TODO
}

Reply:
Also, I think the conditions read better when the variable is put first, e.g. len >= 128...